src/Http/Action/Security/LogInAction.php

```diff
@@ -30,6 +30,9 @@
      */
     private $twig;
 
+    /** @var string */
+    private $sso;
```

Comment:
This field is never used, can we remove it, or revert all of the changes to this file?

Reply:
Removed